### DISCUSSION TITLE
[ags bar]: how to remove those rounded decorations?

### DISCUSSION THREAD

**Discussion by ClSlaid** [end-4/dots-hyprland#248] [Q&A] [answered]
My monitor have no rounded corners, and I'd like to fill all pixels of the monitor with my window.

Sadly, I'm not familiar with frontend and I'd like to know how can I remove those rounded corners?

Just clear all black surroundings in the corners, and make the bar a perfect rectangle.

**Comment by NextStep-IM** [comment, marked as answer]
Are you talking about the top left and top right  corners? If so you can just comment these lines in config.js:
```
forMonitors((id) => Corner(id, 'top left')),    
forMonitors((id) => Corner(id, 'top right')),   
forMonitors((id) => Corner(id, 'bottom left')), 
forMonitors((id) => Corner(id, 'bottom right')),


forMonitors(BarCornerTopleft);  
forMonitors(BarCornerTopright); 

```

**Reply by ClSlaid** [reply]
Thank you, perfectly worked!

**Reply by alok-old** [reply]
Won't this get overwritten when doing `./install.sh` again to update

**Reply by clsty** [reply]
Yes it will get overwritten.

You may write a script, which run `./install.sh` first and then do your custom modification on the overwritten `config.js`.

Or, if @end-4 is willing to provide an option for it in `user_options.js`, use that file after it's done.

**Reply by end-4** [reply]
added in 7926e8f8406a59adba64f8ebf0ad960302086f87

**Reply by alok-old** [reply]
@end-4 Sorry if I am bothering too much, can you do it like this when you are free
```js
    ...(userOptions.appearance.fakeScreenRounding ? [
        forMonitors((id) => Corner(id, 'top left')),
        forMonitors((id) => Corner(id, 'top right')),
        forMonitors((id) => Corner(id, 'bottom left')),
        forMonitors((id) => Corner(id, 'bottom right')),
    ] : []),
    forMonitors(BarCornerTopleft),
    forMonitors(BarCornerTopright),
```
This way only the fake curve on the edges will be affected and not the other one, also I think it looks better this way.

**Reply by end-4** [reply]
done in 4da0d41ee80fe19782617ab596aac6d73aeacd88
